Source/IGListCollectionView.m

```diff
@@ -11,25 +11,6 @@
 
 @implementation IGListCollectionView
 
-- (void)commonInit {
-    self.backgroundColor = [UIColor whiteColor];
-    self.alwaysBounceVertical = YES;
-}
-
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
-    if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        [self commonInit];
```

Comment:
Can we move all of the `commonInit` into here and just get rid of `initWithCoder:`? I think its nice having a default white bg, saves a lot of boilerplate.

Thoughts @jessesquires?